### PR TITLE
1st solution

### PR DESCRIPTION
**Pull request by obrazi3**

### CODE REVIEW

File: Symbols.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        ///and count the maximum number 
+        ///of identical consecutive symbols in it
+        ///</summary>
+        ///<param name="symbols"> 
+        ///The string in which we will count symbols
+        ///</param>
+        ///<returns> the maximum number 
+        ///of identical consecutive symbols in the string
+        ///</returns>
+        public int getMaxCount(string symbols)
```
Comment (IgorKheidorov) [inline Symbols.cs]: Code style violation

File: Symbols.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                    }
+                    else if (currentSymbols > maxSymbols)
+                    {
+                        maxSymbols = currentSymbols;
+                        currentSymbols = 1;
+                    }
+                }
+
+                if (currentSymbols > maxSymbols)
+                    maxSymbols = currentSymbols;
```
Comment (IgorKheidorov) [inline Symbols.cs]: code style

File: Program.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+
```
Comment (IgorKheidorov) [inline Program.cs]: Empty line

File: Symbols.cs [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+{
+    ///<summary>
+    ///This class contains a method 
+    ///for counting the number 
+    ///of identical consecutive characters in a string
+    ///</summary>
+    class Symbols 
+    {
+        ///<summary>
+        /// This method takes a string
```
Comment (AlexanderSoroka) [inline Symbols.cs]: Too much line breaks

File: Symbols.cs [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        ///<param name="symbols"> 
+        ///The string in which we will count symbols
+        ///</param>
+        ///<returns> 
+        ///The maximum number 
+        ///of identical consecutive symbols in the string
+        ///</returns>
+        public int getMaxCount(string symbols)
+        {
+            if (symbols == "")
```
Comment (AlexanderSoroka) [inline Symbols.cs]: empty?

File: Program.cs [hunk 6]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Symbols str = new Symbols();
```
Comment (AlexanderSoroka) [inline Program.cs]: inappropriate variable name

File: Symbols.cs [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+        ///</summary>
+        ///<param name="symbols">The string in which we will count symbols</param>
+        ///<returns>The maximum number of identical consecutive symbols in the string</returns>
+        public int getMaxCount(string symbols)
+        {
+            if (symbols.Length == 0)
+            {
+                return 0;
+            }
+            else
```
Comment (AlexanderSoroka) [inline Symbols.cs]: do not use else for such big block especially in case when method returns from is statement

File: Symbols.cs [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+    ///</summary>
+    class Symbols 
+    {
+        ///<summary>
+        ///This method takes a string and counts the maximum number 
+        ///of identical consecutive characters in it
+        ///</summary>
+        ///<param name="symbols">The string in which we will count symbols</param>
+        ///<returns>The maximum number of identical consecutive symbols in the string</returns>
+        public int getMaxCount(string symbols)
```
Comment (IgorKheidorov) [inline Symbols.cs]: Code style violation, use upper case for methods name

File: Symbols.cs [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+        {
+            if (symbols.Length != 0)
+            {
+                int currentSymbols = 1;
```
Comment (IgorKheidorov) [inline Symbols.cs]: currentSymbolsCount better

File: Program.cs [hunk 10]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Symbols symbols = new Symbols();
+            int maxSymbols = symbols.getMaxCount(args[0]);
```
Comment (IgorKheidorov) [inline Program.cs]: maxSymbolsCount

File: Symbols.cs [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+                if (currentSymbols > maxSymbols)
+                {
+                    maxSymbols = currentSymbols;
+                }
+
+                return maxSymbols;
+            }
+            else
+            {
+                return 0;
```
Comment (IgorKheidorov) [inline Symbols.cs]: Why 0? how this wrong situation is processed?

File: Symbols.cs [hunk 12]
```diff
@@ -0,0 +1,47 @@
+﻿namespace Task1
+{
+    ///<summary>
+    ///This class contains a method for working with a string
+    ///</summary>
+    class Symbols
```
Comment (IgorKheidorov) [inline Symbols.cs]: Bad class name

File: Count.cs [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+        ///of identical consecutive characters in it
+        ///</summary>
+        ///<param name="symbols">The string in which we will count symbols</param>
+        ///<returns>The maximum number of identical consecutive symbols in the string</returns>
+        public int GetMaxCount(string symbols)
+        {
+            int currentSymbolsCount = 1;
+            int maxSymbolsCount = 0;
+
+            if (symbols.Length != 0)
```
Comment (AlexanderSoroka) [inline Count.cs]: Early return 0 if symbols.Length == 0

File: Count.cs [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+        ///This method takes a string and counts the maximum number 
+        ///of identical consecutive characters in it
+        ///</summary>
+        ///<param name="symbols">The string in which we will count symbols</param>
+        ///<returns>The maximum number of identical consecutive symbols in the string</returns>
+        public int GetMaxCount(string symbols)
+        {
+            int maxSymbolsCount = 0;
+
+            if (symbols.Length != 0)
```
Comment (AlexanderSoroka) [inline Count.cs]: use early return instead so long if statement

**Pull request merged**